python/cuml/benchmark/ci_benchmark.py

```diff
@@ -185,6 +185,7 @@ def make_bench_configs(long_config):
                         help='The OS type to include in reports')
     parser.add_argument('--report_machine_name', type=str, default="",
                         help='The machine name to include in reports')
+    parser.add_argument('--n-reps', type=int, default=3)
```

Comment:
```suggestion
    parser.add_argument('--n_reps', type=int, default=3)
```

I think that might be a nicer name for argument consistency

Reply:
ah, I lost control of the args i this file to the CUDA people! ;)
I think hyphens are more standard in python, but you're right that the rest of this utility uses underscores, so I'll update